src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String INFORMATION_SEPARATOR = ",";
+    private static final String SUPPLY_OPERATION = "supply";
+    private static final String BUYING_OPERATION = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        int buy = 0;
+        int supply = 0;
+        int resultOperation = 0;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
```

Comment:
please use try catch with resources https://github.com/mate-academy/jv-work-with-file/pull/867

Reply:
please do not mark conversation as resolved, the mentor should mark it after checking the fix.